_manual/pull_request.md

```diff
@@ -5,6 +5,10 @@ manual: cont
 order: 30
 ---
 
-_...to be written..._
+## Some rules
```

Comment:
'rules' sound too harsh IMHO.

Reply:
advices ?

Reply:
Suggestions? I do not know, just do not want people feel to restricted when not absolute necessary.

Reply:
Agreed with "suggestions". Done in #30.